src/components/SideMenu.jsx

```diff
@@ -7,8 +7,13 @@ import { twMerge } from 'tailwind-merge';
 export default function SideMenu({ children, className, ...props }) {
   const [open, setOpen] = useState(true);
 
+  let width = 'w-48';
```

Comment:
@s1ckret I think it is better to put such things in separate functions, for example: const `width = () => open ? 'w-48' : 'w-8'`, and in general avoid unnecessary declaration of variables.

Reply:
Also in your case conditions in the form of negations are more difficult to perceive, it is better to do it this way:
```ts
let width = 'w-8';
 if (open) {
    width = 'w-48';
  }
```
So we declare a default value and then change it if something changes.